service-front/module/Application/src/Controller/IndexController.php

```diff
@@ -165,4 +166,46 @@ public function abandonFlowAction(): ViewModel
 
         return $view->setTemplate('application/pages/abandoned_flow');
     }
+
+    public function healthCheckAction(): ViewModel
+    {
+        $view = new ViewModel();
+
+        $view->setVariable('status', json_encode([
+            'OK' => true
+        ]));
+
+        return $view->setTemplate('application/pages/healthcheck/healthcheck');
```

Comment:
Need to set the response content type to JSON:
```suggestion
        $this->getResponse()->getHeaders()->addHeaderLine('Content-Type', 'application/json');
        return $view->setTemplate('application/pages/healthcheck/healthcheck');
```